.github/workflows/wheel.yaml

```
@@ -18,14 +30,25 @@ jobs:
     env:
       CIBW_TEST_COMMAND: python -c "import numcodecs"
       CIBW_BUILD: "cp310-* cp311-* cp312-* cp313-*"
-      CIBW_SKIP: "pp* *-musllinux_* *win32 *_i686 *_s390x"
+      CIBW_ARCHS_MACOS: native
```

Think we want to make sure we are building for `x86_64` (native I believe) and `arm64` (for M1,  M2, etc.). Both we already build for. So it would be good to continue supporting them

```suggestion
      CIBW_ARCHS_MACOS: "x86_64 arm64"
```